script/input/1/mainnet-staging.json

```diff
@@ -0,0 +1,14 @@
+{
+    "admin": "0x6F3066538A648b9CFad0679DF0a7e40882A23AA4",
+    "allocatorVault": "0x8E20650287635aE6e20ce38EcD3E795919D52354",
+    "almProxy": "0xC29D06ce81137E6B3C3DC090713636d81600a347",
+    "cctpTokenMessenger": "0xBd3fa81B58Ba92a82136038B25aDec7066af3155",
+    "dai": "0x6B175474E89094C44Da98b954EedeAC495271d0F",
+    "daiUsds": "0x3225737a9Bbb6473CB4a45b7244ACa2BeFdB276A",
```

Comment:
I think we use a mock dai usds for staging. Not sure where this address is, but let's find it.

Reply:
Current staging controller is using this contract and it will work the same way, should be fine:

```
cast call 0xcc0c5ADF6649256d3cE6084eCf94AF5D01440b6C 'daiUsds()(address)'
0x3225737a9Bbb6473CB4a45b7244ACa2BeFdB276A
```

We'll just swap 1:1 between USDS and DAI, the mock usdsJoin and vault is what was important